chapters/methodology/theory.tex

```diff
@@ -127,15 +149,18 @@ \subsection{The $\Omega$ Methods}
                            d\Omega}.
 \label{eq:omega_basic}
 \end{equation}
-%
+% should the d \Omega have a hat? I'm not sure...
+
 The $\Omega$-flux is a hybridization of the adjoint scalar flux and the contributon
 flux. It is both a normalized contributon flux and a forward-weighted
 adjoint flux. Theoretically, it should inherit some of the advantages of
 each of the traditional
 adjoint and the contributon fluxes. Because it maintains dimensionality of the
 traditional adjoint scalar flux, it can be used in place of the standard adjoint
 scalar flux in both
-CADIS and FW-CADIS variance reduction parameter generation.
+CADIS and FW-CADIS variance reduction parameter generation. 
+This means that nothing about the CADIS and FW-CADIS implementations need to 
```

Comment:
and I understand about not wanting to make it seem trivial. On the other hand, highlighting that something isn't too burdensome to adopt is a super valuable thing re impact of your work :)